linked list.java

```diff
@@ -0,0 +1,33 @@
+import java.util.*;  
+public class LinkedList2{  
```

Comment:
Please place in the respective folder, also follow the naming convention, refer to readme file.